mmv1/third_party/terraform/website/docs/guides/version_7_upgrade.html.markdown

```diff
@@ -238,3 +238,11 @@ Remove `template.containers.depends_on` from your configuration after upgrade.
 The default value for `disable_on_destroy` has been changed to `false`. The previous default (`true`) created a risk of unintended service disruptions, as destroying a single `google_project_service` resource would disable the API for the entire project.
 
 Now, destroying the resource will only remove it from Terraform's state and leave the service enabled. To disable a service when the resource is destroyed, you must now make an explicit decision by setting `disable_on_destroy = true`.
+
+## Resource: `google_memorystore_instance`
+
+ `allow_fewer_zones_Deployment` has been removed as user configurable but will be set to true at creation
```

Comment:
"removed as user-configurable" could be interpreted to mean "it is now output-only" but actually we're just fully removing it. I'm not sure about the reason I've proposed; that's not generally a reason to remove a field. But I'm not sure if there's a better way to put it?

```suggestion
 `allow_fewer_zones_deployment` has been removed because it isn't user-configurable.
```